docs/walkthrough/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: default
+title: Deploying Promitor, Prometheus, and Grafana on an AKS Cluster
+---
+
+In this walkthrough, we'll set up a basic monitoring solution with Promitor, Prometheus, and Grafana. In order to have a resource to monitor, we'll create a Service Bus queue and add load to the queue with Service Bus Explorer. We'll deploy Promitor, Prometheus, and Grafana to a Kubernetes cluster using Helm, and explain how each of these services connects and how to see output. We'll also walk through setting up a Prometheus alert and Grafana dashboard to show alerting and visualization.
+
+## Prerequisites
+
+- The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the Kubernetes command-line tool. It can also be installed via the Azure CLI with `az aks install-cli`.
+- [Helm](https://helm.sh/docs/using_helm/#installing-the-helm-client), a Kubernetes deployment manager
+- [Service Bus Explorer](https://github.com/paolosalvatori/ServiceBusExplorer/releases)
+
+## Create a Resource Group
+
+
+```bash
+az group create --name PromitorRG --location eastus
+```
+
+Output: 
+```json
+{ 
+  "id": "/subscriptions/<guid-subscription-id>/resourceGroups/PromitorRG",
+  "...": "..."
+}
+```
+
+## Create a Service Principal
+
+Use the resource group creation output to add a scope to your service principal:
+
+```bash
+az ad sp create-for-rbac \
+  --role="Monitoring Reader" \
+  --scopes="/subscriptions/<guid-subscription-id>/resourceGroups/PromitorRG"
```

Comment:
Just a thought - Should we scope this on the RG or the Service Bus namespace? Maybe we should be as secure as possible?

Not a blocker though.

Reply:
I figure RG so that the user could more easily add other resources to monitor if they wanted.